dev-python/authlib/authlib-1.5.2.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10,11,12,13} )
+
+DISTUTILS_USE_PEP517="setuptools"
+
+inherit distutils-r1 pypi
+
+DESCRIPTION="A Python library in building OAuth and OpenID Connect servers and clients."
+
+HOMEPAGE="https://authlib.org/"
```

Comment:
Please always include GitHub repo and PyPI project page. These things are helpful when you need to check stuff quickly.